tests/kernel/blktests.pm

```diff
@@ -17,6 +17,26 @@ use testapi;
 use utils;
 use repo_tools 'add_qa_head_repo';
 
+#can accept tags: none, all, specific
+sub _test_devices {
+    my ($devices) = @_;
+
+    if ($devices eq 'all') {
+        $devices = script_output('lsblk -d -n -p -o NAME');
+        $devices = (join ' ', split ' ', $devices);
+    } elsif ($devices eq 'specific') {
+        die('BLK_DEVICE_ONLY: specific - not implemented');
```

Comment:
Is it placeholder for current test or for blktests itself?

Reply:
I would like to be able to provide eventually an option that the test could list devices to tests like /dev/1 /dev/2 etc. so
specify:/dev/1,/dev/2

Reply:
this is done a it differently now.